### ISSUE TITLE
Getting an Unable to Write entry at installation

### ISSUE THREAD

**Issue by jjsjjs** [asikart/userxtd#22]
Hello, i was trying your component but i'm getting an Unable to Write entry at installation Error.
Any idea?

And is this still not for production sites?

Thanks !

**Comment by asika32764** [comment]
It should be able to use on production site since 1.0.0, but it seems there are new bugs after 3.3.6.

I'll try to reproduce these bugs when I have time.'

About your problem, could you please open Debug mode in global configuration then get a screenshot for me?

**Comment by jjsjjs** [comment]
Hello Simon,

I made a clean joomla 3.3.6 install on easyphp local server.

When i install userxtd i get an Error : Unable to write entry.

I have set debug mode on but i do not get any debug or error information 
on screen(front end). Only white screen.

In Backend:
Category = OK
Fields = White screen
Profiles =  White screen
Users = White screen

So allready at install something goes wrong, maybe that is why the rest 
does not work ok.

Thanks for helping.

Kind regards,
Jerry

Simon Asika schreef op 22-1-2015 om 5:53:
